ui/eureka/eureka.js

```diff
@@ -3621,7 +3622,9 @@ class EurekaTracker {
     // Adds field note drops, name, id & rarity of those
     if (this.zoneInfo.treatNMsAsSkirmishes && this.options.EnrichedSTQ && nm.fieldNotes) {
       for (const note of fieldNotesList) {
-        if (note.id === nm.fieldNotes)
+        if (typeof nm.fieldNotes === 'string')
+          nmNote = [note];
+        if (note.id in nmNote)
           enriched.innerHTML = `#${note.id}: ${note.shortName} ${gRarityIcon.repeat(note.rarity)}`;
```

Comment:
I was pondering whether to add it on the same line or to make it smaller, not sure yet. It can get kinda long but the user can choose to disable enriched content anyway, so maybe we should go with 2 on the same line ? only happens for a single skirmish anyway

Reply:
Multiple lines is probably ok!